workspace.json

```diff
@@ -619,6 +673,46 @@
           }
         }
       }
+    },
+    "plugins-scully-plugin-md-replace-route": {
+      "root": "libs/plugins/scully-plugin-md-replace-route",
+      "sourceRoot": "libs/plugins/scully-plugin-md-replace-route/src",
+      "projectType": "library",
+      "schematics": {},
+      "architect": {
+        "lint": {
+          "builder": "@nrwl/linter:lint",
+          "options": {
+            "linter": "eslint",
+            "tsConfig": [
+              "libs/plugins/scully-plugin-md-replace-route/tsconfig.lib.json",
+              "libs/plugins/scully-plugin-md-replace-route/tsconfig.spec.json"
+            ],
+            "exclude": [
+              "**/node_modules/**",
+              "!libs/plugins/scully-plugin-md-replace-route/**/*"
+            ]
+          }
+        },
+        "test": {
+          "builder": "@nrwl/jest:jest",
+          "options": {
+            "jestConfig": "libs/plugins/scully-plugin-md-replace-route/jest.config.js",
+            "tsConfig": "libs/plugins/scully-plugin-md-replace-route/tsconfig.spec.json",
+            "passWithNoTests": true
+          }
+        },
+        "build": {
+          "builder": "@nrwl/node:package",
+          "options": {
+            "outputPath": "dist/libs/plugins/scully-plugin-md-replace-route",
```

Comment:
this is one of the places

Reply:
ok I'll check this out

Reply:
I added the ``` --importPath @scullyio/scully-plugin-md-replace-route``` but still see the same line 708 in this file... I think I am a little confused about this...

Reply:
I do manually change those things. Otherwise, we will end up in a situation where we have different imports for internal use inside our monirepo

Reply:
OK sounds good